lua/telescope/builtin/internal.lua

```diff
@@ -31,12 +31,9 @@ end
 
 local internal = {}
 
--- TODO: What the heck should we do for accepting this.
---  vim.fn.setreg("+", "nnoremap $TODO :lua require('telescope.builtin').<whatever>()<CR>")
--- TODO: Can we just do the names instead?
 internal.builtin = function(opts)
-  opts.path_display = utils.get_default(opts.path_display, "hidden")
-  opts.ignore_filename = utils.get_default(opts.ignore_filename, true)
+  opts = opts or {}
+  opts.ignore_filename = utils.get_default(opts.ignore_filename, false)
```

Comment:
@Conni2461, Is this what you meant?

Reply:
No . Nil them before passing them into the pickers you open.

So they dont mess up things ...

Reply:
Also you dont need to do `opts = opts or {}` in telescope builtins. We do this automatically so it doesnt error if we forget it.

Here is how we do it

https://github.com/nvim-telescope/telescope.nvim/blob/5f37fbfa837dfee7ecd30f388b271f4a71c0a9e0/lua/telescope/builtin/internal.lua#L1219-L1231